packages/automl/frontend/src/app/components/run-results/PipelinePreparingState.scss

```
@@ -0,0 +1,26 @@
+.automl-pipeline-preparing__content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  max-width: 420px;
+  padding: var(--pf-t--global--spacer--xl);
+}
+
+.automl-pipeline-preparing__spinner {
+  --pf-v6-c-spinner--diameter: 4rem;
+  margin-bottom: var(--pf-t--global--spacer--lg);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)